kairon/shared/actions/data_objects.py

```
@@ -1051,4 +1053,70 @@ class ParallelActionConfig(Auditlog):
+    """
+
+    expression_type = StringField(required=True, choices=["cron", "epoch"])
+    schedule = StringField(required=True)
+    timezone = StringField(required=True)
+
+    def validate(self, clean=True):
+        if clean:
+            self.clean()
+
+        if not self.timezone or not self.timezone.strip():
+            raise ValidationError("timezone is required for all schedules!")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)